### ISSUE TITLE
[Bug] Switching from "select tokens" to "measure distance" breaks all token control actions

### ISSUE THREAD

**Issue by Someone2345** [caewok/fvtt-elevation-ruler#107]
FVTT v12.327
D&D5e 3.2.1
Active modules 38 
Update: tested with only fvtt-elevation-ruler enabled and managed to easily replicate.

All token controls become unresponsive after switch from dragging tokens with the select tokens tool, and then switching to the measure distance tool.  After changing to the measure distance tool: 

- Character sheets no longer open on double click of tokens.
- Right click to view token options does not work.
- Tokens can no longer be dragged with the token select tool. 
- Measurement tool does not work
- (with splatter module installed, add blood splatter does not work (as no token is selected). 

Only way to regain control is to refresh the web page. 

![image](https://github.com/caewok/fvtt-elevation-ruler/assets/75526961/d48a5808-0a05-4072-bd9f-8e6b23d329ef)

![Screenshot 2024-06-27 222419](https://github.com/caewok/fvtt-elevation-ruler/assets/75526961/3e162c0f-efa5-4a49-b59e-88955a86ca36)

![Screenshot 2024-06-27 222432](https://github.com/caewok/fvtt-elevation-ruler/assets/75526961/3f7f61e2-d470-406d-84b6-c06b54b9b215)

[All modules appart from fvtt-elevation-ruler disabled - same results, same error messages in the log]
![image](https://github.com/caewok/fvtt-elevation-ruler/assets/75526961/8e0363a5-f664-4001-ba05-21a07fefb17f)

**Comment by Duncan-Henry** [comment]
I've also experienced this issue when playing on the PF2E system. It was easy to replicate whenever someone would try to measure a distance without having a token selected.

**Comment by Rapunzel77** [comment]
When using the measure tool I get this stack trace:

```
Uncaught TypeError: Cannot read properties of undefined (reading 'BACKGROUND_ELEVATION')
    at Ruler._addWaypoint (Ruler.js:161:107)
    at 🎁CONFIG.Canvas.rulerClass.prototype._addWaypoint#0 (libWrapper-wrapper.js:187:52)
    at Ruler._startMeasurement (foundry.js:36051:10)
    at Ruler._onDragStart (foundry.js:36512:10)
    at 🎁call_wrapped [as call_wrapped] (libWrapper-wrapper.js:507:22)
    at Ruler._onDragStart (Ruler.js:561:10)
    at 🎁CONFIG.Canvas.rulerClass.prototype._onDragStart#0 (libWrapper-wrapper.js:187:52)
    at #onDragLeftStart (foundry.js:25733:34)
    at MouseInteractionManager.callback (foundry.js:30691:17)
    at #handleDragStart (foundry.js:31053:15)
    at #handlePointerMove (foundry.js:31031:43)
    at vh.notifyListeners (EventBoundary.ts:1464:33)
    at vh.notifyTarget (EventBoundary.ts:658:14)
    at vh.propagate (EventBoundary.ts:310:18)
    at vh.dispatchEvent (EventBoundary.ts:210:14)
    at vh.mapPointerMove (EventBoundary.ts:815:64)
    at vh.mapEvent (EventBoundary.ts:231:28)
    at ra.onPointerMove (EventSystem.ts:396:31)
```

**Comment by Nolat** [comment]
Yes, there is no BACKGROUND_ELEVATION flags in the const.js file, so it breaks the ruler

**Comment by caewok** [comment]
Fixed in v0.9.6. Thanks!
